src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarClassification.tsx

```diff
@@ -36,3 +70,80 @@ export const ArtworkSidebarClassificationFragmentContainer = createFragmentConta
     }
   `
 )
+
+const ClassificationLink = styled.span`
+  cursor: pointer;
+  text-decoration: underline;
+`
+
+const ClassificationDetails = () => {
+  const classificationOptions = [
```

Comment:
Also this is not very kosher. I think we might want to get those from MP since there we already have a class that lists all this strings. But I was not sure if the best thing is to add it to artwork or do a separate call for AttributionClasses so just put it inline for now.